.github/workflows/ci.yml

```diff
@@ -37,4 +37,5 @@ jobs:
       - uses: nrwl/nx-set-shas@v4
 
       # Nx Affected runs only tasks affected by the changes in this PR/commit. Learn more: https://nx.dev/ci/features/affected
-      - run: pnpm exec nx affected -t lint build unit-test integration-test e2e-ci
+      # - run: pnpm exec nx run-many -t build unit-test integration-test e2e-ci
+      - run: pnpm exec nx run-many -t lint
```

Comment:
this is scary

Reply:
Understand. I just wanted to reduce the noise. But i get that this can get forgotten and the CI is green :D

Let me switch the order and comment back in.

Reply:
I un-scared you here: https://github.com/Coly010/ng-rspack-build/pull/105/commits/16728b9b17af8fe3a43b5095cc12756f678d7a9c